internal/pkg/commands/loglevel.go

```diff
@@ -271,7 +274,8 @@ func (options *ListLogLevelsOpts) Execute(args []string) error {
 
 	// Already error checked during option processing
 	host, port, _ := splitEndpoint(GlobalConfig.KvStore, defaultKvHost, defaultKvPort)
-	cm := config.NewConfigManager(client, supportedKvStoreType, host, port, int(GlobalConfig.KvStoreConfig.Timeout.Seconds()))
+	GlobalConfig.KvStore = net.JoinHostPort(host, strconv.Itoa(port))
```

Comment:
Don't set the GlobalConfig.KvStore again here. Better directly use net.JoinHostPort(host, strconv.Itoa(port)) as argument of config.NewConfigManager method